hyperopt/atpe.py

```diff
@@ -488,8 +489,10 @@ def createHyperparameterConfigForHyperoptDomain(domain):
             return data
         elif domain.name == "randint":
             data = {"type": "number"}
-            max = domain.pos_args[0]._obj
-            data["max"] = max
+            low = domain.pos_args[0]._obj
+            high = domain.pos_args[1]._obj
+            data["min"] = 0 if high is None else low
+            data["max"] = high or low
```

Comment:
shouldn't this be
`data["min"] =  0 if low is None else low`
`data["max"] = high`

and in this case high will be required parameter and low will be optional, currently in this code, if I am not wrong, low is treated as mandatory argument and it means upper bound (high) in case low is not given... and it makes less sense.